Header header
# /// Measurement
int16 value
